front/app/modules/commercial/project_folders/permissions/rules.ts

```diff
@@ -1,32 +1,9 @@
 import { IProjectFolderData } from './../services/projectFolders';
-import {
-  definePermissionRule,
-  IRouteItem,
-} from 'services/permissions/permissions';
-import { isProjectFolderModerator, moderatesFolder } from './roles';
+import { definePermissionRule } from 'services/permissions/permissions';
+import { moderatesFolder } from './roles';
 import { isAdmin } from 'services/permissions/roles';
-import {
-  canAccessRoute,
-  isAdminRoute,
-  MODERATOR_ROUTES,
-} from 'services/permissions/rules/routePermissions';
-import { IUser } from 'services/users';
-import { IAppConfigurationData } from 'services/appConfiguration';
-
-const canUserAccessAdminFolderRoute = (
```

Comment:
I have no idea how this was working. From what I could tell, this was firing on every route change.

This is the only other place in the app we do `definePermissionRule('route', 'access'`